<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vAnalyserForm</class>
 <widget class="QDialog" name="vAnalyserForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>422</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/options/code-context.png</normaloff>:/options/code-context.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Наименование</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit">
       <property name="text">
        <string>Noname</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Обучение</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Метод обучения</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox">
           <item>
            <property name="text">
             <string>NEAT</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>NEVA</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>libFANN</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Общие параметры</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Продолжать обучение</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox_2">
              <item>
               <property name="text">
                <string>До первого неуспеха</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>До окончания выборки</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Максимальный размер выборки</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBox">
              <property name="maximum">
               <number>10000</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Критерии обучения методом обратного распространения ошибки</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Обрабатывать файл</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox_5">
              <item>
               <property name="text">
                <string>Полностью</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>До достижения % ошибки</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <item>
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>До достижения % ошибки</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="doubleSpinBox"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>26</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_fann">
      <attribute name="title">
       <string>FANN</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QSpinBox" name="spinBox_2">
             <property name="minimum">
              <number>1</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="verticalSlider">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QGraphicsView" name="graphicsView"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QSlider" name="horizontalSlider">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_3">
           <property name="minimum">
            <number>1</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Функция активации (внешняя)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_3">
           <item>
            <property name="text">
             <string>FANN_LINEAR</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>FANN_THRESHOLD</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>FANN_THRESHOLD_SYMMETRIC</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>FANN_SIGMOID</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>FANN_SIGMOID_STEPWISE</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>FANN_SIGMOID_SYMMETRIC</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>FANN_SIGMOID_SYMMETRIC_STEPWISE</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>FANN_GAUSSIAN</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>FANN_GAUSSIAN_SYMMETRIC</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>FANN_GAUSSIAN_STEPWISE</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>FANN_ELLIOT</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>FANN_ELLIOT_SYMMETRIC</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>FANN_LINEAR_PIECE</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>FANN_LINEAR_PIECE_SYMMETRIC</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>FANN_SIN_SYMMETRIC</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>FANN_COS_SYMMETRIC</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>FANN_SIN</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>FANN_COS</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Функци активации (внутренняя)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_4">
           <item>
            <property name="text">
             <string>FANN_LINEAR</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>FANN_THRESHOLD</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>FANN_THRESHOLD_SYMMETRIC</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>FANN_SIGMOID</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>FANN_SIGMOID_STEPWISE</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>FANN_SIGMOID_SYMMETRIC</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>FANN_SIGMOID_SYMMETRIC_STEPWISE</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>FANN_GAUSSIAN</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>FANN_GAUSSIAN_SYMMETRIC</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>FANN_GAUSSIAN_STEPWISE</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>FANN_ELLIOT</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>FANN_ELLIOT_SYMMETRIC</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>FANN_LINEAR_PIECE</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>FANN_LINEAR_PIECE_SYMMETRIC</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>FANN_SIN_SYMMETRIC</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>FANN_COS_SYMMETRIC</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>FANN_SIN</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>FANN_COS</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Входные данные</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QTreeWidget" name="treeWidget">
         <column>
          <property name="text">
           <string>Данные</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Выходные данные</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QTreeWidget" name="treeWidget_2">
         <column>
          <property name="text">
           <string>Наименование</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Данные</string>
          </property>
         </column>
         <item>
          <property name="text">
           <string>Результаты</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Получатели данных</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>vAnalyserForm</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>vAnalyserForm</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
